docs/reference/action_list.rst

```diff
@@ -284,7 +284,38 @@ Configuring the default ordering column can be achieved by overriding the
 
 .. note::
 
-    **TODO**: how to sort by multiple fields (this might be a separate recipe?)
+    For UI reason, it's not possible to sort by multiple fields. However, this behavior can be simulate by
+    adding some default orders in the ``configureQuery()`` method. The following example is using
+    ``SonataAdminBundle`` with ``SonataDoctrineORMAdminBundle``::
```

Comment:
Since you use `::` here, you should indent the code block below, shouldn't you?

Reply:
Indeed, fixed